tests/unit/switch_utils.c

```diff
@@ -80,6 +80,12 @@ FST_TEST_BEGIN(b64)
 }
 FST_TEST_END()
 
+FST_TEST_BEGIN(is_file_path)
+{
+	switch_bool_t b = switch_is_file_path("{av_record_audio_only=true");
```

Comment:
Test this with multiple inner bracket as well. e.g.
```
{av_record_audio_only=true,future_json_var='{key=value}'"
```

Reply:
good point; I've added new test